examples/NewRelic.OpenTelemetry/ConsoleCore/Program.cs

```diff
@@ -12,20 +12,27 @@ class Program
         private const string ActivitySourceName = "NewRelic.OpenTelemetryExporter.SampleConsoleCoreApp";
         private static readonly ActivitySource SampleActivitySource = new ActivitySource(ActivitySourceName);
 
-        // Set these values for yourself
-        private const string MyNewRelicInsightsInsertApiKey = "<YOUR_NR_INSIGHTS_INSERT_API_KEY_HERE>";
-        private const string MyServiceName = "SampleConsoleCoreApp";
+        // Set these values by environment variable
+        private static readonly string MyNewRelicInsightsInsertApiKey = Environment.GetEnvironmentVariable("NR_API_KEY");
+        private static readonly string MyServiceName = Environment.GetEnvironmentVariable("NR_SAMPLE_APP_NAME") ?? "SampleConsoleCoreApp";
```

Comment:
Instead of environment variables, let's make our usage of appsettings.json consistent across our examples. See:

https://github.com/newrelic/newrelic-telemetry-sdk-dotnet/blob/281223149ad73ac94f87fd7eafc41fa278bdbb1d/examples/NewRelic.OpenTelemetry/AspNetCore/Startup.cs#L35-L36

For a console app you'll have to manually set up configuration. Something like:

```csharp
var builder = new ConfigurationBuilder()
    .AddJsonFile($"appsettings.json", true, true)
    .AddEnvironmentVariables();

var config = builder.Build();
```

Reply:
Sounds good.  I made the change, let me know what you think.